src/custumers/api.py

```diff
@@ -0,0 +1,80 @@
+from flask_openapi3 import APIBlueprint, Tag
+
+
+from src.custumers.models import Custumer
+from src.custumers.schemas import CustumerDeleteSchema, CustumersPostSchema
+
+custumers_api = APIBlueprint(
+     "/custumers",
+     __name__,
+     url_prefix="/api",
+     abp_tags=[Tag(name="Custumers", description="Random custumers")],
+     doc_ui=True,
+ )
+
+
+@custumers_api.get("/custumers")
+def get_custumers():
+    return [{"id": custumer.id, "name": custumer.name, "cep": custumer.cep, "email": custumer.email} for custumer in Custumer.query.all()]
+
+
+@custumers_api.post("/custumers")
+def create_custumer(body: CustumersPostSchema):
+    print("#############", body)
+    custumer = Custumer(
+        name=body.name,
+        email=body.email,
+        cep=body.cep,
+        uf=body.uf,
+        city=body.city,
+        street=body.street,
+        number=body.number,
+        complement=body.complement).save()
+    return {"id": custumer.id, "name": custumer.name}
+
+
+@custumers_api.delete("/custumers")
+def delete_custumer(body: CustumerDeleteSchema):
+    email = body.email
+    if not email:
+        return {"error": "Email is required to delete a customer"}, 400
+
+    custumer = Custumer.query.filter_by(email=email).first()
+    if not custumer:
+        return {"error": "Customer not found"}, 404
+
+    custumer.delete()
+    return {"message": "Customer deleted successfully"}
+
+
+@custumers_api.put("/custumers")
+def update_custumer(body: CustumersPostSchema):
+    email = body.email
+    print(f"Received request to update customer with email: {email}")
+
+    custumer = Custumer.query.filter_by(email=email).first()
+    if not custumer:
+        print("Customer not found")
+        return {"error": "Customer not found"}, 404
+
+    print(f"Updating customer: {custumer}")
+
+    try:
```

Comment:
@PedroTeixeira-dev Great idea, we can make better use of try except throughout our code. Well done 😉 .

Reply:
:)